app/routes/classifications_jitter.py

```diff
@@ -0,0 +1,86 @@
+import os
+import random
+
+import redis
+from flask import render_template, redirect, url_for
+from rq import Connection, Queue
+from rq.job import Job
+from torchvision import transforms
+
+from app import app
+from app.forms.color_jitter_form import ClassificationColorJitterForm
+from config import Configuration
+from ml.classification_utils import classify_image, fetch_image  #
+
+config = Configuration()
+
+
+@app.route('/classifications_jitter', methods=['GET', 'POST'])
+def classificationsJitter():
+    """
+    API for selecting a model and an image and running a
```

Comment:
Please, update the docstring (the Jitter transformation should be mentioned) :book: